PerformanceMetrics/CMakeLists.txt

```diff
@@ -90,4 +91,6 @@ if(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_YOUTUBE)
     write_config( PLUGINS PerfMetricsYouTube )
 endif()
 
-
+if (PLUGIN_PERFORMANCEMETRICS_RDKBROWSER)
```

Comment:
By adding a copy of write_config( PLUGINS PerfMetricsWebKitBrowser ) you now  write the config file twice. As mentioned PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER must be true otherwise none of the monitoring would make sense, and then if you select (PLUGIN_PERFORMANCEMETRICS_RDKBROWSER you would write PerfMetricsWebKitBrowser here as wel as earlier in the CMake file. It is better to do that only once and use some logic in the if clause (although I think you can remove the new one and leave the below one as is, if PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER is not true it also has no use monitoring on classname as the plugin will not be in the image, so if it is true you always want to generate the config, only the content will be dependent on the classname flag)

Reply:
I removed the duplicate codes. And used OR for PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_CLASSNAME.